panels/doll_panel.py

```diff
@@ -360,3 +360,6 @@ def get_controls(self):
   def display_message(self, message):
     """Prints the message with a prefix (to differentiate it from what the user types)."""
     print('> ' + message)
+
+  def display_status(self, data):
```

Comment:
@dawnho you're going to need to implement this as described [here](https://github.com/wearhere/spacecontrol/commit/a0cd01a80d80a92657b1e9b47da79ef519f062dc#diff-04c6e90faac2675aa89e2176d2eec7d8L97), example [here](https://github.com/wearhere/spacecontrol/blob/a0cd01a/panels/keyboard_panel.py#L151)